app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
```

Comment:
Consider clearing the `Person.people` dictionary at the beginning of this function to avoid retaining data from previous calls, which could lead to incorrect results.